vllm/attention/selector.py

```diff
@@ -240,10 +152,8 @@ def global_force_attn_backend_context_manager(
     attn_backend: AttentionBackendEnum,
```

Comment:
I think this might not work if the "forcing" manager is outside the vLLM config creation. Can we just always use the config and fully remove this card manager? Or at least use the env variable?